src/chrome/content/rules/Steamrep.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Steamrep.com">
+  <target host="steamrep.com" />
+  <target host="www.steamrep.com" />
+
+  <rule from="^http://(www\.)?steamrep\.com/" to="https://www.steamrep.com/" />
```

Comment:
Ah, I see what you mean now. https://www.steamrep.com redirects to
http://steamrep.com, but then that would get rewritten by HTTPS
Everywhere also and you'd wind up on https://steamrep.com. Okay, I'm
convinced your way is better. :thumbsup: